doc/netplan.md

```diff
@@ -437,6 +437,24 @@ similar to ``gateway*``, and ``search:`` is a list of search domains.
             dhcp6: true
             optional-addresses: [ ipv4-ll, dhcp6 ]
 
+``activation-mode`` (scalar) – since **0.102**
+
+:    Allows specifying the management policy of the selected interface. By
+     default, netplan brings up any configured interface if possible. Using the
+     ``activation-mode`` setting users can override that behavior by either
+     specifying "manual", to hand over control over the interface state to the
+     administrator or (for networkd backend *only*) "off" to force the link
```

Comment:
Just some formatting here, I think we usually emphasize the values as a code-style block.

```suggestion
     specifying ``manual``, to hand over control over the interface state to the
     administrator or (for networkd backend *only*) ``off`` to force the link
```